### PR TITLE
add spacing/size dependency on screen size

### PR DESCRIPTION
**Pull request by Jonathansoufer**

### CODE REVIEW

**Comment by obernardovieira** [comment]
Can you please add a video or screenshot?

**Comment by Jonathansoufer** [comment]
WhatsApp Image 2021-06-21 at 12 03 47

File: src/views/communities/Stories.tsx [hunk 1]
```diff
@@ -109,23 +109,22 @@ export default function Stories() {
                 showsHorizontalScrollIndicator={false}
                 contentContainerStyle={{ padding: 18 }}
             >
-                {userAddress.length > 0 &&
+                {/* {userAddress.length > 0 &&
```
Comment (obernardovieira) [inline src/views/communities/Stories.tsx:112]: Only logged users in a valid (aka accepted and public) community (beneficiary or managers) should be able to add stories. Why are you changing this?

**Comment by Jonathansoufer** [comment]
Sorry, that was a test left forgotten on the code. Already fixed it.

**Review by obernardovieira — APPROVED** [review]

**Pull request merged**